helios-tools/src/main/java/com/spotify/helios/cli/command/RollingUpdateCommand.java

```diff
@@ -184,27 +186,35 @@ protected int runWithJobId(final Namespace options, final HeliosClient client,
       return 1;
     }
 
+    final RolloutOptions optionsFromJob = job.getRolloutOptions();
+    final Integer actualParallelism = parallelism != null
```

Comment:
consider extracting a function for this repeated ternary statement - I assume you don't want `firstNonNull` as it is ok if the second param is null?

Reply:
done